packages/libra-rpc-graphql/README.md

```diff
@@ -1,7 +1,7 @@
 # `@shopify/libra-rpc-graphql`
 
 [![Build Status](https://travis-ci.com/Shopify/libra-web-tools.svg?branch=master)](https://travis-ci.com/github/Shopify/libra-web-tools)
-[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Flibra-rpc-graphql.svg)](https://badge.fury.io/js/%40shopify%2Flibra-rpc-graphql.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/libra-rpc-graphql.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/libra-rpc-graphql.svg)
```

Comment:
this package shouldn't really ever be used in the browser, so i'm removing this web-focused bundle size badge.